index/score.go

```diff
@@ -20,13 +20,13 @@ import (
 	"math"
 	"strings"
 
+	"github.com/go-enry/go-enry/v2"
 	"github.com/sourcegraph/zoekt"
 	"github.com/sourcegraph/zoekt/internal/ctags"
 )
 
 const (
-	ScoreOffset     = 10_000_000
-	ScoreOffsetBM25 = 1_000_000_000
```

Comment:
In addition to the file part of the tiebreaker, I removed the repo tiebreaker from BM25 to simplify the scoring. It wasn't doing much, because it's really rare for BM25 to produce ties. The only thing this makes worse: if you have two completely identical files, but one is from a fork that's not touched often.

For repo ranking, I'd like to explore an alternative approach similar to the 'personalized repos' work, where we focus on diversifying the repo set and adjusting it through reranking.